cms_monPerf

```diff
@@ -10,62 +10,58 @@
 # 4.      paging load, and
 # 5.      network utilization.
 
-max() {
-  printf "%d\n" "${@}" | sort -rn | head -1
-}
-
 NCPU=$(awk '/processor/ {nr++} END{print nr}' /proc/cpuinfo)
-LOADAVG=$(< /proc/loadavg)
-MEMINFO=$(< /proc/meminfo)
 
-LOAD_ARR=(${LOADAVG})
-LOAD5=$(echo "scale=0;${LOAD_ARR[1]}*100/${NCPU}" | bc -l)
+LOAD5=$(awk '{print int($2*100/'${NCPU}')}' /proc/loadavg)
```

Comment:
maybe it would be cleaner to use the -v option?   
`LOAD5=$(awk -v ncpu=${NCPU} '{print int($2*100/ncpu)}' /proc/loadavg)`

Reply:
Yes, that's much cleaner!